src/Routers/AggregateRouter.js

```diff
@@ -7,57 +7,51 @@ import UsersRouter   from './UsersRouter';
 const ALLOWED_KEYS = [
   'where',
   'distinct',
-  'project',
+  'addFields',
```

Comment:
Can we make a list of `PIPELINE_KEYS` and `BASE_KEYS` and concat them in `ALLOWED_KEYS` for readability?